R/tm_g_association.R

```diff
@@ -359,22 +359,45 @@ srv_tm_g_association <- function(id,
         )
       })
 
+      # helper function to format variable name
+      foo <- function(x) {
+        if (is.numeric(ANL[[x]]) && log_transformation) {
+          varname_w_label(x, ANL, prefix = "Log of ")
+        } else {
+          varname_w_label(x, ANL)
+        }
+      }
+      new_title <-
+        if (association) {
+          switch(as.character(length(vars_names)),
+            "0" = sprintf("Value distribution for %s", ref_cl_lbl),
+            "1" = sprintf(
+              "Association between %s and %s",
+              ref_cl_lbl,
+              foo(vars_names)
+            ),
+            sprintf(
+              "Associations between %s and: %s",
+              ref_cl_lbl,
+              paste(lapply(vars_names, foo), collapse = ", ")
+            )
+          )
+        } else {
+          switch(as.character(length(vars_names)),
+            "0" = sprintf("Value distribution for %s", ref_cl_lbl),
+            sprintf(
```

Comment:
So line 97 -99 I think means you are not allowed multiple reference values (i.e. if you put muiltiple = TRUE ln examples on line 48 it won't work so I don't think you need this?

Reply:
I'm not sure I follow. This is not for multiple references, but for multiple associated values, which is allowed in the current implementation.

Reply:
hmm  - maybe I misunderstood, let me look more closely

Reply:
yup my bad - ignore this 👍